pkg/checkpoint/checkpoint_restore.go

```diff
@@ -228,14 +229,14 @@ func CRImportCheckpoint(ctx context.Context, runtime *libpod.Runtime, restoreOpt
 	containerConfig := container.Config()
 	ctrName := ctrConfig.Name
 	if containerConfig.Name != ctrName {
-		return nil, errors.Errorf("Name of restored container (%s) does not match requested name (%s)", containerConfig.Name, ctrName)
+		return nil, fmt.Errorf("name of restored container (%s) does not match requested name (%s)", containerConfig.Name, ctrName)
 	}
 
 	if !newName {
 		// Only check ID for a restore with the same name.
 		// Using -n to request a new name for the restored container, will also create a new ID
 		if containerConfig.ID != ctrID {
-			return nil, errors.Errorf("ID of restored container (%s) does not match requested ID (%s)", containerConfig.ID, ctrID)
+			return nil, fmt.Errorf("id of restored container (%s) does not match requested ID (%s)", containerConfig.ID, ctrID)
```

Comment:
```suggestion
			return nil, fmt.Errorf("ID of restored container (%s) does not match requested ID (%s)", containerConfig.ID, ctrID)
```

"id" vs "ID" in the same error message confuses me